pkg/metrics/general/serve_stats.go

```diff
@@ -34,6 +39,79 @@ func roundOff(num uint64) float64 {
 	return math.Round(x*10) / 10
 }
 
+// ServeInfo provides information about the system such as OS info, uptime, boot time, etc.
+func ServeInfo(ctx context.Context, cpuChannel chan AggregatedMetrics) error {
+	info, err := host.InfoWithContext(ctx)
+	if err != nil {
+		return err
+	}
+
+	hostInfo := [][]string{
+		{"Hostname", info.Hostname},
+		{"Up Time", utils.SecondsToHuman(int(info.Uptime))},
+		{"Boot Time", utils.GetDateFromUnix(int64(info.BootTime))},
```

Comment:
It looks like the boot time is in Unix epoch seconds, whereas `GetDateFromUnix` expects Unix epoch *milliseconds*. This is why it showed my boot time as `20 Jan 70` in the screenshot before lol.

This fixes it, although I'm not sure if it's standard across systems:
```suggestion
		{"Boot Time", utils.GetDateFromUnix(int64(info.BootTime * 1000))},
```

Reply:
Oh oops I don't know how I missed that